src/basic/string-util.c

```diff
@@ -703,7 +705,17 @@ char *strip_tab_ansi(char **ibuf, size_t *_isz, size_t highlight[2]) {
         assert(ibuf);
         assert(*ibuf);
 
-        /* Strips ANSI color and replaces TABs by 8 spaces */
+        /* This does three things:
+         *
+         * 1. Replaces TABs by 8 spaces
+         * 2. Strips ANSI color sequences, i.e. CSI '[' … 'm' sequences
+         * 3. Strips ANSI operating system sequences, i.e. CSO '[' … BEL sequences
```

Comment:
I think it's `CSO ']' ... BEL sequences`, at least from reading the code...

Reply:
Actually, ']' is already part of the CSO sequence itself, hence the existing comment is broken in even more ways, as a strict reading would hence mean ESC ']' '[' … BEL... which is of course bogus...